### ISSUE TITLE
loss definition is different from the paper

### ISSUE THREAD

**Issue by buaaJeremyduan** [cogaplex-bts/bts#13]
In your code, the a scale-invariant error is "self.silog_loss = tf.sqrt(tf.reduce_mean(d ** 2) - 0.85 * (tf.reduce_mean(d) ** 2)) * 10.0",  It is different from the Eq.(4) of the paper because there is no "tf.sqrt"

**Comment by cogaplex-bts** [comment]
Thanks for pointing this.
We had intensive experiments with various settings and missed it in the current version of the paper.
We will update it accordingly.
Thanks.
